packages/ndla-editor/package.json

```diff
@@ -28,10 +28,10 @@
     "@ndla/core": "^0.7.3",
     "@ndla/icons": "^1.4.1",
     "@ndla/modal": "^1.1.20",
-    "@types/react-beautiful-dnd": "^11.0.3",
+    "@types/react-beautiful-dnd": "^10.1.5",
```

Comment:
Var det meininga å nedgradere denne?

Reply:
Ja og nei. Featuren jeg bruker for å bumpe pakker påsto at dette var den nyeste versjonen, men det var det jo tydeligvis ikke.